README.md

```diff
@@ -27,7 +27,10 @@ namespace ChartGettingStarted
         public MainPage()
         {
             InitializeComponent();           
+            Grid grid = new Grid();
             SfCartesianChart chart = new SfCartesianChart(); 
+            grid.Add(chart);
+            this.Content = grid; 
```

Comment:
Remove the grid as we discussed in xaml and c#

Reply:
Removed